addons/bus/static/src/bus_core_service.js

```diff
@@ -0,0 +1,14 @@
+/* @odoo-module */
+
+import { registry } from "@web/core/registry";
+
+export const busCoreService = {
+    dependencies: ["bus_service"],
+    start(env, services) {
+        services.bus_service.subscribe("forceUpdateChannels", () => {
+            services.bus_service.forceUpdateChannels();
+        });
```

Comment:
Probably ok, but this has own DDoS potential if there are many users refreshing at once.